Project Management/PI Planning/24PI3 Planning/PI Planning Preparation by Squad/Zowe Doc Squad - 24PI3 Objectives.md

```diff
@@ -11,18 +11,12 @@ Key areas to address in doc:
 2. Angular 12 upgrade
 3. Other day 1, new v3 features
 
-### Create content for migrating from V2 to V3
+### Review content for migrating from V2 to V3
```

Comment:
Maybe update to create AND review content? I am still actively updating/creating content for V3